config/engines.yml

```diff
@@ -184,6 +184,7 @@ sonar-java:
   channels:
     beta: codeclimate/codeclimate-sonar-java:beta
   description: SonarLint for Java.
+  memory: 2_048_000_000
```

Comment:
What do you think about changing this key to `memory_limit`?

Reply:
From the explanation below, this is not the `limit`... maybe `minimum_required_memory`?

Reply:
I think `memory_limit` is probably fine for the registry. The fact that we can give it more memory either via the CLI or in production can be a behind the scenes detail.

Reply:
@dblandin Made it `required_memory`. Would give it another shot?

Reply:
The option this affects it the upper limit of memory a container can use, not necessarily guaranteeing that amount of memory will be available. That's more of a clustering/scheduling issue.

What do you think of `minimum_memory_limit` or simply `memory_limit` as an override key to the default, but knowing that the _allowed_ memory limit might be increased through other means, like a repo-override or CLI option.

Reply:
`minimum_memory_limit` makes sense to me! Adjusted.

Reply:
@dblandin Do you mind giving another shot?